tableauserverclient/server/endpoint/workbooks_endpoint.py

```diff
@@ -104,7 +104,7 @@ def update_connection(self, workbook_item, connection_item):
         server_response = self.put_request(url, update_req)
         connection = ConnectionItem.from_response(server_response.content, self.parent_srv.namespace)[0]
 
-        logger.info('Updated workbook item (ID: {0} & connection item {1}'.format(workbook_item.id,
+        logger.info('Updated workbook item (ID: {0} & connection item {1})'.format(workbook_item.id,
                                                                                   connection_item.id))
```

Comment:
Looks like the linter is complaining here, the `c` should align with the `w` (one more space in front)